packages/shared/src/contexts/SettingsContext.tsx

```diff
@@ -197,6 +198,8 @@ export const SettingsContextProvider = ({
           ...settings,
           sidebarExpanded: !settings.sidebarExpanded,
         }),
+      onSidebarExpanded: (value: boolean) =>
+        setSettings({ ...settings, sidebarExpanded: value }),
```

Comment:
We have to introduce a function to set it on what value we really expect it to be rather than toggling. The issue is that, the default value might not match what is the default value defined in GB, so toggling would just reset the value to the same thing.

Reply:
What does GB stands for here? I guess it's not Great Britain, neither Group Buy nor Giga Byte 😅

Reply:
Hahah Growthbook @kopancek